README.md

```diff
@@ -38,7 +38,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
```

Comment:
🔄 Consider updating the version tag to a more descriptive format if possible (e.g., v1.0.0-alpha, v1.0.0-beta) to indicate the stability or type of release. This helps in understanding the release context better.